comptests/ha/test_restart_pod.py

```diff
@@ -125,11 +121,35 @@ def test_datapod_status_online_after_replicas(self):
         assert_utils.assert_false(resp[0], f"Failed to delete pod {pod_name} by making replicas=0")
         LOGGER.info("Step 1: Successfully shutdown/deleted pod %s by making replicas=0", pod_name)
         self.deployment_name = resp[1]
-        self.restore_pod = True
         self.restore_method = common_const.RESTORE_SCALE_REPLICAS
+        self.restore_pod = True
 
-        #TODO: Step-2 | Getting multiple events for pod delete operation - CORTX-28560
-        #TODO: Step-3 | Pod Online events are not seeing in the health monitor log - CORTX-28867
+        LOGGER.info("Step 2: Check pod failed alert in health monitor log")
+        node_obj = self.ha_comp_obj.get_ha_node_object(self.node_master_list[0])
+        resp_dict = self.ha_comp_obj.get_ha_log_prop(node_obj, common_const.HA_SHUTDOWN_LOGS[2],
+                                                     kvalue=1, health_monitor=True)
+        source_list = resp_dict['source']
+        resource_type_list = resp_dict['resource_type']
+        resource_status_list = resp_dict['resource_status']
+        generation_id_list = resp_dict['generation_id']
+        failed_node_id = resp_dict['node_id']
+        failed_resource_id = resp_dict['resource_id']
+        for index, (data1, data2, data3) in enumerate(zip(source_list, resource_type_list,
```

Comment:
Create a common function in lib and call them in each test.

Reply:
Created a local method as does only assert.